[W-11723892] Adding VPN risk docs (#620)
mulesoft/docs-runtime-manager: modules/ROOT/pages/vpn-high-availability.adoc

image:logo-server-disabled.png[xref="deployment-strategies.adoc#anypoint-platform-pce-deployments", title="Anypoint Platform PCE"]
image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]

> Your VPN connection consists of two VPN tunnels, each terminating in a different Availability Zone, to provide increased availability to your VPC. During a mandatory upgrade, AWS may briefly disable one of the two tunnels of your VPN connection. When you configure your customer gateway, it's therefore important that you configure both tunnels for redundancy. When one tunnel becomes unavailable (for example, down for maintenance), network traffic is automatically routed to the other available tunnel for that specific VPN connection. When AWS performs updates on one VPN tunnel, they set a lower outbound multi-exit discriminator (MED) value on the other tunnel to ensure the failover. Accordingly, AWS Site-to-Site VPN https://aws.amazon.com/vpn/site-to-site-vpn-sla/[SLA] also requires two VPN endpoints for each connection. You will run a significantly higher risk of MuleSoft service downtime during the AWS auto patching window if you do not have redundant Anypoint VPN tunnels configuration.

Reviewer: ```suggestion
Your VPN connection consists of two VPN tunnels, each terminating in a different Availability Zone, to provide increased availability to your VPC. During a mandatory upgrade, AWS may briefly disable one of the two tunnels of your VPN connection. When you configure your customer gateway, configure both tunnels for redundancy. When one tunnel becomes unavailable (for example, down for maintenance), AWS automatically routes network traffic to the other available tunnel for that specific VPN connection. When AWS performs updates on one VPN tunnel, they set a lower outbound multi-exit discriminator (MED) value on the other tunnel to ensure the failover.  AWS Site-to-Site VPN https://aws.amazon.com/vpn/site-to-site-vpn-sla/[SLA] also requires two VPN endpoints for each connection.  Redundant Anypoint VPN tunnel configuration prevents a significantly higher risk of MuleSoft service downtime during the AWS auto patching window.
```

A MuleSoft Virtual Private Gateway (VGW) supports one Anypoint VPC association, but it supports up to 10 VPN connections.
You can locate your VPN Gateways in the same data center, or in different physical locations.

> To eliminate all single points of failure, the two customer gateways should be in different data centers, and preferably in different cities. A good choice is to you your Disaster Recovery location for the second gateway.

Reviewer: ```suggestion
To eliminate all single points of failure, the two customer gateways should be in different data centers, and preferably in different cities. Ideally, use your Disaster Recovery location for the second gateway.
```
